app/controllers/api/v1/partners_controller.rb

```diff
@@ -5,13 +5,16 @@ class Api::V1::PartnersController < ApiController
   def create
     return head :forbidden unless api_key_valid?
 
-    partner = Partner.invite!(email: partner_params[:email],
+    partner = Partner.new(
```

Comment:
Rather than new, we *may* want to do `find_or_create_by`. I know we've had diaper banks tell us that the people working at a non-profit who had the account on the partner system left and they need to add another from their side. Might be nice to allow diaper banks to themselves add multiple people from a partner organization to partnerbase?